tutorials/variational-inference/01_variational-inference.jmd

```diff
@@ -32,44 +31,32 @@ Random.seed!(42);
 The Normal-(Inverse)Gamma conjugate model is defined by the following generative process
 
 \begin{align}
-    s &\sim \mathrm{InverseGamma}(2, 3) \\
-    m &\sim \mathcal{N}(0, s) \\
+    s &\sim \mathrm{InverseGamma}(2, 3) \\\\
+    m &\sim \mathcal{N}(0, s) \\\\
     x_i &\overset{\text{i.i.d.}}{=} \mathcal{N}(m, s), \quad i = 1, \dots, n
 \end{align}
 
 Recall that *conjugate* refers to the fact that we can obtain a closed-form expression for the posterior. Of course one wouldn't use something like variational inference for a conjugate model, but it's useful as a simple demonstration as we can compare the result to the true posterior.
 
 First we generate some synthetic data, define the `Turing.Model` and instantiate the model on the data:
 
-
-```julia
-# generate data, n = 2000
+```julia; results = "hidden"
+# generate data
 x = randn(2000);
 ```
 
-
-
-
 ```julia
 @model model(x) = begin
     s ~ InverseGamma(2, 3)
     m ~ Normal(0.0, sqrt(s))
     for i = 1:length(x)
         x[i] ~ Normal(m, sqrt(s))
     end
-end
+end;
```

Comment:
Maybe better to use `results = "hidden"` here as well? Or just leave it as it is for now?

Reply:
So it actually seems like this is not intended? https://github.com/JunoLab/Weave.jl/issues/386

If so, we can just remove all the `results = "hidden"` as this was the reason why I added them.

Reply:
This issue seems to be about a different problem though, doesn't it? It doesn't mention `results = "hidden"`, which according to the documentation is intended to hide the output?

Reply:
I added `results = "hidden"` because `;` wasn't respected. If `;` is respected, then we none of the `results = "hidden"` I added are needed.

Reply:
Made a PR with a fix, so I'll remove these `results = "hidden"` and wait for new relase.

Reply:
Great!

I'm still surprised that `results = "hidden"` doesn't actually suppress the output even though it is mentioned in the docs in this way.

Reply:
I generally think the semicolons are a bit ugly but let's stick with them if that's the only way to make it work reliably :shrug:

Reply:
> I'm still surprised that results = "hidden" doesn't actually suppress the output even though it is mentioned in the docs in this way.

Wat. No it _does_ What did I say? :confused: 

> I generally think the semicolons are a bit ugly but let's stick with them if that's the only way to make it work reliably 

Yeaah, let's for now :+1: